usr/lib/common/loadsave.c

```diff
@@ -864,20 +936,14 @@ CK_RV load_private_token_objects_old(STDLL_TokData_t *tokdata)
     CK_RV rc;
     size_t read_size;
 
-    sprintf(iname, "%s/%s/%s", tokdata->data_store, PK_LITE_OBJ_DIR,
-            PK_LITE_OBJ_IDX);
-
-    fp1 = fopen(iname, "r");
+    fp1 = open_token_object_index(iname, sizeof(iname), tokdata, "r");
     if (!fp1)
         return CKR_OK;          // no token objects
 
     while (fgets(tmp, 50, fp1)) {
```

Comment:
where does this "50" come from ?
Can't this be replaced by some kind of sizeof(tmp) ?

Reply:
Changed this.  Maybe we should think about a different dimension for tmp than PATH_MAX?

Reply:
There are 4 places in OCK with fgets(tmp, 50 ...). The token object names have just 8 chars, but when reviewing the token migration tool, we changed this to PATH_MAX. Why not do the same here?